verified-structures/jellyfish-merkle-rocksdb/src/main/java/com/bloxbean/cardano/vds/jmt/rocksdb/RocksDbJmtSchema.java

```diff
@@ -0,0 +1,117 @@
+package com.bloxbean.cardano.vds.jmt.rocksdb;
+
+import com.bloxbean.cardano.vds.rocksdb.namespace.NamespaceOptions;
+
+/**
+ * Schema definition for JMT RocksDB storage with namespace support.
+ *
+ * <p>Supports dual independent prefixes:
+ * <ul>
+ *   <li>Column Family Prefix: Controls which CF to use (e.g., "account_nodes_jmt")</li>
+ *   <li>Key Prefix: Always present (1 byte), prepended to all keys</li>
+ * </ul>
+ */
+final class RocksDbJmtSchema {
+    // Base column family names (no _shared suffix needed)
+    static final String CF_NODES = "nodes_jmt";
+    static final String CF_VALUES = "values_jmt";
+    static final String CF_ROOTS = "roots_jmt";
+    static final String CF_STALE = "stale_jmt";
+    static final String CF_NODES_BY_VERSION = "nodes_by_ver_jmt";
+    static final String CF_VALUES_BY_VERSION = "values_by_ver_jmt";
+
+    static final byte[] LATEST_ROOT_KEY = new byte[]{'J', 'M', 'T', '_', 'L', 'A', 'T', 'E', 'S', 'T'};
+    static final byte[] LATEST_VERSION_KEY = new byte[]{'J', 'M', 'T', '_', 'V', 'E', 'R'};
+
+    /**
+     * Generates column family names and key prefix from namespace options.
+     *
+     * @param options namespace configuration (null uses defaults)
+     * @return column families with CF names and key prefix
+     */
+    static ColumnFamilies columnFamilies(NamespaceOptions options) {
+        NamespaceOptions opts = (options != null) ? options : NamespaceOptions.defaults();
+
+        // Simple: just add CF prefix if present
+        String cfPrefix = opts.columnFamilyPrefix().isEmpty()
+            ? ""
+            : opts.columnFamilyPrefix() + "_";
+
+        return new ColumnFamilies(
+                cfPrefix + CF_NODES,
+                cfPrefix + CF_VALUES,
+                cfPrefix + CF_ROOTS,
+                cfPrefix + CF_STALE,
+                cfPrefix + CF_NODES_BY_VERSION,
+                cfPrefix + CF_VALUES_BY_VERSION,
+                opts.keyPrefix()  // Always present!
+        );
+    }
+
+    /**
+     * Legacy method for backward compatibility.
+     *
+     * @deprecated Use {@link #columnFamilies(NamespaceOptions)} instead
+     */
+    @Deprecated
+    static ColumnFamilies columnFamilies(String namespace) {
+        if (namespace == null || namespace.isBlank()) {
+            return columnFamilies(NamespaceOptions.defaults());
+        } else {
+            return columnFamilies(NamespaceOptions.columnFamily(namespace.trim()));
+        }
+    }
```

Comment:
is this used? this is the first version of this lib maybe the deprecation can be just removed?

Reply:
Thanks @nemo83 . Yes, we need to remove any unused or any methods marked as deprecated.